# Feb - Ana Conesa

!!! info "Event Details"

     **Date/Time:**

    Thursday, February 29th, 2024 :material-clock: 11:00am - 1:30pm PT

    **Location:**

    :material-video: UBC, Michael Smith Laboratories, MSL 102

    :material-video:  SFU Big Data Hub, room ASB 10900 (Live Stream Location)

**Featured Speaker**: [Dr. Ana Conesa](http://conesalab.org/)

**Talk Title:** Third-Generation Long-Read Transcriptome Sequencing for Unraveling Transcriptome Complexity

<!-- ![type:video](https://www.youtube.com/embed/<CODE>) -->

**Affiliation:**

- Spanish National Research Council, Research Professor
- University of Florida, Courtesy Professor
- Biobam Bioinformatics, co-founder

**Bio:**

Ana Conesa is Research Professor at the Spanish National Research Council (CSIC) and Courtesy Professor at the University of Florida. She is a member Spanish Royal Academy of Engineer, Fellow of the International Society for Computational Biology, honorary member of the Spanish Society of Bioinformatics and Computational Biology and member of the Board of Directors of the International Society for Computational Biology. She directs the CSIC network for Computational Biology and is co-founder of Biobam Bioinformatics, a start-up that provides bioinformatics tools for biologists. Ana Conesaâ€™s lab is interested in understanding functional aspects of gene expression at the genome-wide level and across different organisms. Her group has developed statistical methods and software tools for transcriptomics analysis and she has pioneered the development of methods for multi-omics integration and long-reads transcriptomics. A strong drive in her research is helping the genomics community to bridge the gap between data and knowledge by creating bioinformatics tools that everybody can use. Some of our popular software tools are Blast2GO, PaintOmics, maSigPro, NOISeq, Qualimap, SQANTI, tappAS, etc that have received over 37,000 citations. She has led multiple EU projects to develop methods for the analysis of the transcriptome, more recently with a focus on the utilization of long read sequencing to characterize transcriptome complexity.

**Abstract:**

Third-generation long-read sequencing technologies offer the potential to sequence entire transcripts and unravel the intricacies of transcriptomes. However, the analysis of long-read transcriptomics (lrRNA-seq) data presents numerous challenges. These challenges encompass distinguishing biological variability from technical noise, accurately predicting transcript models, providing precise estimates of transcript expression levels and differential expression, and elucidating the biological significance of isoform diversity. In my presentation, I will showcase the research conducted in my laboratory that addresses these challenges. Moreover, I will discuss how, through the implementation of appropriate experimental techniques and bioinformatics approaches, lrRNA-seq has the capability to unveil fresh insights into the biology of the transcriptome.

---

**Trainee Speaker:** Dollina Dodani

<!-- **Affiliation:** <TODO> -->

<!-- **Talk Title**: <TODO>  -->
